CefSharp.OutOfProcess.Wpf.HwndHost/ChromiumWebBrowser.cs

```diff
@@ -319,6 +320,18 @@ public ChromiumWebBrowser(OutOfProcessHost host, string initialAddress = null)
             UseLayoutRounding = true;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ChromiumWebBrowser"/> instance.
+        /// </summary>
+        /// <param name="host">Out of process host</param>
+        /// <param name="initialAddress">address to load initially</param>
+        /// <param name="requestContextPreferences">requestContextPreferences to set</param>
+        public ChromiumWebBrowser(OutOfProcessHost host, string initialAddress = null, IDictionary<string, object> requestContextPreferences = null)
```

Comment:
Was there a reason for an additional constructor? Looks like `requestContextPreferences` could just be added as an optional arg to the existing constructor.

Reply:
done